src/NewDataList/NewDataList.js

```diff
@@ -124,5 +124,14 @@ module.exports = kind({
 			Spotlight.spot(fv);
 			return true;
 		}
+	},
+	isVisibleChild : function(control){
```

Comment:
Could you also include comment blocks, identifying this as a private event (similar to `eventIsFromScrollingChild`)?